resources/windows/wix/project.wxs.erb

```diff
@@ -20,6 +20,10 @@
       <!--  We are discussing a MAINT PR to move the componentfilegroup to seperate file
             This would allow project to specify additional fragments if needed -->
 
+    <UI>
+      <UIRef Id="WixUI_MainInstallDialog"/>
```

Comment:
@jpogran Will work at a solution for this tomorrow.
What do you think of the alternative suggestion of putting in the componentgroups.wxs in the project specific area ?

/cc @melissa @McdonaldSeanp